packages/json/src/browser/json-client-contribution.ts

```diff
@@ -92,6 +109,7 @@ export class JsonClientContribution extends BaseLanguageClientContribution {
         // handle content request
         languageClient.onRequest('vscode/content', async (uriPath: string) => {
             const uri = new URI(uriPath);
+            // @ts-ignore
```

Comment:
@ndoschek what are you attempting to ignore?
I think if it's necessary, adding an additional comment here for the reason might be worthwhile.

Reply:
Oh sorry, that was not intended, removed that line again.